examples/slab2.py

```diff
@@ -0,0 +1,39 @@
+"""
+Slab2 - A Comprehensive Subduction Zone Geometry Model
+======================================================
+
+Slab2 is a three-dimensional compilation of global subduction geometries, separated
+into regional models for each major subduction zone.
+More information at the
+`USGS <https://www.sciencebase.gov/catalog/item/5aa1b00ee4b0b1c392e86467>`__ website.
+"""
+
+import rockhound as rh
+import matplotlib.pyplot as plt
+import cmocean
+import cartopy.crs as ccrs
+
+from rockhound.slab2 import ZONES
+
+plt.figure(figsize=(10, 5))
+ax = plt.axes(projection=ccrs.Robinson())
+
+for zone in ZONES:
+    grid = rh.fetch_slab2(zone)
+
+    pc = grid.depth.plot.pcolormesh(
+        cmap=cmocean.cm.thermal_r,
+        ax=ax,
+        transform=ccrs.PlateCarree(),
+        add_colorbar=False,
+        vmin=-701160.33935547,
+        vmax=-1399.69277382,
```

Comment:
Instead of hardcoding the min and max values I would compute them inside the example.